test/e2e/helpers/visu/PageTester.ts

```diff
@@ -16,18 +16,11 @@
 import { FitType, LoadOptions } from '../../../../src/component/options';
 import { ElementHandle } from 'puppeteer';
 import { BpmnQuerySelectors } from '../../../../src/component/registry/bpmn-elements-registry';
-import { copyFileSync, loadBpmnContentForUrlQueryParam } from '../../../helpers/file-helper';
-import { log } from '../test-utils';
-
-export enum BpmnLoadMethod {
-  QueryParam = 'query param',
-  Url = 'url',
-}
 
 export interface TargetedPage {
   pageFileName: string;
```

Comment:
⚠️ as mentionned in https://github.com/process-analytics/bpmn-visualization-js/pull/966#pullrequestreview-559583648, I suggest we rename this property

Reply:
I don't know howx to find a better name.

For me, the file name (of the page) is just the name (ex: 'index') without the extension (ex: '.html')

Reply:
To avoid spending too much time on this topic, I propose one of the following
- keep the current `pageFileName` and add TSDoc to mention that the extension should not be passed
- rename into `name` and add TSDoc as well if it is not clear enough